common/models/order.ts

```diff
@@ -72,7 +72,7 @@ export class OrderEntity implements Order {
       order.orderId,
       order.createdAt,
       order.servedAt,
-      order.items,
+      order.items as WithId<ItemEntity>[],
```

Comment:
このままでは item が ItemEntity にならずに変換されてしまいます
```suggestion
      order.items.map((item) => ItemEntity.fromItem(item))
```